addons/payment/wizards/payment_link_wizard_views.xml

```diff
@@ -5,10 +5,14 @@
         <field name="model">payment.link.wizard</field>
         <field name="arch" type="xml">
             <form string="Generate Payment Link">
+                <group attr="{'invisible':[('partner_email', '=', False)]}">
```

Comment:
shouldn't it be the opposite?
```suggestion
                <group attrs="{'invisible':[('partner_email', '!=', False)]}">
```
seemed to work because you used `attr` instead of `attrs` :stuck_out_tongue:

Reply:
@dbo-odoo Indeed, I did test directly in the frontend template, forgot to change it in the final one.